mesa_geo/geospace.py

```diff
@@ -48,6 +44,21 @@ def __init__(self, crs="epsg:3857"):
         self.idx = index.Index()
         self.idx.agents = {}
 
+    @property
+    def crs(self):
+        return self._crs
+
+    @property
+    def transformer(self):
+        return self._transformer
+
+    @crs.setter
+    def crs(self, crs):
+        self._crs = pyproj.CRS.from_user_input(crs)
+        self._transformer = pyproj.Transformer.from_crs(
+            crs_from=self.crs, crs_to="epsg:4326", always_xy=True
+        )
+
     def add_agents(self, agents):
```

Comment:
I just noticed that the API is a bit confusing. `agents` can either be a list of `GeoAgent` or a single `GeoAgent`. *This concern is out of the scope of this PR*, but I think we should just simplify it so that it accepts only a list of `GeoAgent`. If people want to add only 1 agent, they can just use a list containing 1 agent. This way, the code block within the `hasattr(agent, "geometry")` is also applied to the case of multiple agents.

Reply:
Would it make sense to be consistent with Mesa and have a  `place_agent()` method for single agent? To add list of agents, perhaps we have an additional `place_agents()` method (or is it more confusing?).

Reply:
Hi, not sure about that idea. Mesa's `place_agent` takes a `pos` attribute, which isn`t really applicable here, so it would be the same function name with different arguments.

Reply:
1 vs many agents probably boils down to which case is used more often by users. My gut feeling would say its more common to add a list of agents so @rht suggestion might be a good idea, but this is also something that could be resolved once we get some feedback from users (generally I wouldn't change to many things that aren't problems right now - even if the alternative ideas would have been a better solution in the first place)

Reply:
> Hi, not sure about that idea. Mesa's `place_agent` takes a `pos` attribute, which isn`t really applicable here, so it would be the same function name with different arguments.

Ah I miss function overloading : )

In terms of 1 vs many agents, I don't have any particular preference.